advanced/Scripts/piholeDebug.sh

```diff
@@ -493,6 +493,12 @@ parse_setup_vars() {
   fi
 }
 
+parse_locale() {
+  echo_current_diagnostic "Locale"
+  local pihole_locale="$(locale)"
```

Comment:
Instead of trying to pass this through the file parser and having to deal with adding in line endings and all that, how about we just dump the output from `locale` straight to `log_write`? Without making some changes this implementation will only log the first line of the `locale` output, (which might be enough, as that's LANG, but we may want to know more).